src/components/Target.vue

```diff
@@ -2,8 +2,8 @@
   <p>
     {{ this.val }}
```

Comment:
This is the line that the val is used. If you only delete the properties, it would throw an error over here.

Reply:
So if you want the delete the val(bus station name), please delete this line too.